src/main/java/org/pixlaunch/module_02/day_04/assignment/ques_01/Solution.java

```diff
@@ -11,7 +11,7 @@ private int findFactorCount(int a) {
         for(int i=1; i<Math.sqrt(a); i++) {
             if (a % i==0) {
                 if(i == a/i) {
-                    count++;
+                    count += 1;
```

Comment:
Bothings means same. Count ++ and Count = count+1

Reply:
please ignore, this is by mistake I am trying to learn github and raised this pull request by mistake. Yes both are same